assets/src/styles/blocks/Covers/layouts/TakeActionCovers.scss

```diff
@@ -22,10 +22,6 @@
   margin: auto;
   margin-bottom: 24px;
 
-  &.action-card {
```

Comment:
This class was only used by the Take Action Boxout but it's no longer the case since that block now uses separate styles